web-stories.php

```diff
@@ -52,7 +52,7 @@
 
 $cdn_version = 'main';
 
-if ( false !== strpos( WEBSTORIES_VERSION, '+' ) ) {
+if ( false === strpos( WEBSTORIES_VERSION, '+' ) ) {
 	$pieces      = explode( '+', WEBSTORIES_VERSION );
```

Comment:
If `WEBSTORIES_VERSION` doesn't contain the string "+", how do we split/explode on it?

Reply:
Lol 🤦 Let me fix that real quick

Reply:
Ah forgot that PHP simply returns the full string if the `+` isn't found. No problem there whatsoever:

https://3v4l.org/VgoRX

Reply:
Should we just remove this line then?

Reply:
One step at a time.

The goal of this section is to fall back to `main` if one is using a development build (which has a commit hash in the version number)

Example:
* I submit a PR to add a new template to the gallery.
* I already added the template images to the `main` folder in the `static-site` branch. 
* Now when I enable that PR on the QA site, I need the correct new template images to be loaded

Yes, it's not ideal, but it has worked well for us so far, and alternative options (that allow removing this code), have been explored in #4752 already.

**tl;dr:**

* I would not remove this entire code for 1.2.1 - just hotfix it.
* #4752 needs a solution in the future

Reply:
I meant removing just the line that calls `explode()`, or am I misunderstanding?

Reply:
Then plugin version like `1.2.3+abcf123` could not be handled

Reply:
"1.2.3+abcf123" would not enter the conditional and `$cdn_version` will remain "main".